Sources/MlemMiddleware/FeedLoaders/User Content Feed Loader/UserContent.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  UserContent.swift
+//  
+//
+//  Created by Eric Andrews on 2024-07-21.
+//
+
+import Foundation
+
+public class UserContent: Hashable, Equatable, FeedLoadable {
+    public typealias FilterType = UserContentFilterType
+    
+    public let wrappedValue: Value
```

Comment:
Not as it currently stands, since `FeedLoadable` is `AnyObject`, which `enum` cannot conform to.

I'm not actually sure exactly why that conformance is in place, but until we have all the `FeedLoader`s implemented I don't want to modify it because *presumably* I put it there for a reason. I've opened #35 to track this.